SECURITY.md

```diff
@@ -55,10 +55,7 @@ Security.insertProviderAt(Conscrypt.newProvider(), 1);
 JDK versions prior to Java 9 do not support ALPN and are either missing AES GCM
 support or have 2% the performance of OpenSSL.
 
-We recommend most users use grpc-netty-shaded, which includes netty-tcnative on
```

Comment:
This paragraph shouldn't move; we need the Conscrypt and Java 9+ discussion here.